src/Microsoft.Azure.ServiceBus/Management/QueueDescription.cs

```diff
@@ -176,186 +168,10 @@ public string ForwardDeadLetteredMessagesTo
-                            new XElement(XName.Get("MaxSizeInMegabytes", ManagementClientConstants.SbNs), XmlConvert.ToString(this.MaxSizeInMB)),
-                            new XElement(XName.Get("RequiresDuplicateDetection", ManagementClientConstants.SbNs), XmlConvert.ToString(this.RequiresDuplicateDetection)),
-                            new XElement(XName.Get("RequiresSession", ManagementClientConstants.SbNs), XmlConvert.ToString(this.RequiresSession)),
-                            this.DefaultMessageTimeToLive != TimeSpan.MaxValue ? new XElement(XName.Get("DefaultMessageTimeToLive", ManagementClientConstants.SbNs), XmlConvert.ToString(this.DefaultMessageTimeToLive)) : null,
-                            new XElement(XName.Get("DeadLetteringOnMessageExpiration", ManagementClientConstants.SbNs), XmlConvert.ToString(this.EnableDeadLetteringOnMessageExpiration)),
-                            this.RequiresDuplicateDetection && this.DuplicateDetectionHistoryTimeWindow != default ?
-                                new XElement(XName.Get("DuplicateDetectionHistoryTimeWindow", ManagementClientConstants.SbNs), XmlConvert.ToString(this.DuplicateDetectionHistoryTimeWindow))
-                                : null,
-                            new XElement(XName.Get("MaxDeliveryCount", ManagementClientConstants.SbNs), XmlConvert.ToString(this.MaxDeliveryCount)),
-                            new XElement(XName.Get("EnableBatchedOperations", ManagementClientConstants.SbNs), XmlConvert.ToString(this.EnableBatchedOperations)),
-                            this.authorizationRules != null ? this.AuthorizationRules.Serialize() : null,
-                            new XElement(XName.Get("Status", ManagementClientConstants.SbNs), this.Status.ToString()),
-                            this.ForwardTo != null ? new XElement(XName.Get("ForwardTo", ManagementClientConstants.SbNs), this.ForwardTo) : null,
-                            this.AutoDeleteOnIdle != TimeSpan.MaxValue ? new XElement(XName.Get("AutoDeleteOnIdle", ManagementClientConstants.SbNs), XmlConvert.ToString(this.AutoDeleteOnIdle)) : null,
-                            new XElement(XName.Get("EnablePartitioning", ManagementClientConstants.SbNs), XmlConvert.ToString(this.EnablePartitioning)),
-                            this.ForwardDeadLetteredMessagesTo != null ? new XElement(XName.Get("ForwardDeadLetteredMessagesTo", ManagementClientConstants.SbNs), this.ForwardDeadLetteredMessagesTo) : null
-                        ))
-                    ));
-
-            return doc;
-        }
 
-        public bool Equals(QueueDescription other)
+        public bool Equals(QueueDescription otherDescription)
         {
-            if (this.Path.Equals(other.Path, StringComparison.OrdinalIgnoreCase)
+            if (otherDescription is QueueDescription other && this.Path.Equals(other.Path, StringComparison.OrdinalIgnoreCase)
```

Comment:
>(otherDescription is QueueDescription other [](start = 15, length = 43)

This is always true :)

Reply:
PR was green on top of your branch 😉

Reply:
> This is always true :)

Let me get to that in a separate issue 😃